### PR TITLE
Updates to "Releasing" section of README - 1734

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://asteele-update-releasing-readme-1734--60f9b557105290003b387cd5.chromatic.com

---

## Description
Closes [#1734](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1734)

## Acceptance criteria
- [ ] Approved by team

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,10 +55,23 @@ This repo follows [semantic versioning](https://semver.org/). Here are some exam
...
 
 **For Design System Team only:**
-If the version numbers have been updated in `main` and you're ready to publish, [draft a new release](https://github.com/department-of-veterans-affairs/component-library/releases) where the tag follows the [`vX.Y.Z` semantic versioning structure](https://semver.org/). This tag should match the version found in `packages/core/package.json`. After creating a new tag, click the Generate release notes button. Verify the release notes look correct, and then click the Publish release button. Once a release is created, an attempt will be made to automatically publish the package(s) to npm using a GitHub action.
+
+1. If you are unsure if a new release should be created, check with the Release Manager and/or the rest of the team first, to make sure it's worth the effort at this time.
+2. Get the version number from `packages/core/package.json`, ensuring it's up-to-date and new.
```
Comment (jamigibbs) [inline README.md:63]: For my laziness, can we just make this `packages/core/package.json` a link to the actual file?  https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/core/package.json#L4

**Comment by Andrew565** [comment]
> * The `ignore-for-release` label seems to be a source of confusion.

I don't disagree, it can be confusing sometimes. I suspect that people are much more likely to see and therefore act on the ignore-for-release instructions in the PR template than if we include a sentence or two about it in the README, though. Unfortunately just one of those things I think we have to continue handling on an as-needed basis, as you can lead a dev to the instructions, but you can't make them read and understand them.

> * Is it worth mentioning when the `core` version should be incremented? In the past it was always just "the last PR merged before a release" so I guess that's just kind of something the Release Manager should be on top of?

I'll try and make this a little more obvious.

**Review by powellkerry — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -55,10 +55,23 @@ This repo follows [semantic versioning](https://semver.org/). Here are some exam
...
+3. From the [repo's homepage](https://github.com/department-of-veterans-affairs/component-library) click on "Releases" in the right-hand sidebar.
+4. Click on the "Draft a new release" button near the top of the page.
+5. Click on the 'Choose a tag' drop-down and type the letter `v` followed by the new "core" version number (should look like `v16.1.0`). The target should remain `main`.
+6. For the release title, type the same thing you entered for the tag (`v{versionNumber}`).
+7. Click on the "Generate release notes" button. If the button is disabled, double-check that the tag/version number is correct and hasn't been released before.
+8. Review the release notes for any typos and/or unneeded notes. Remember that these release notes are intended for public use, so they should be professional in their tone and appearance.
+9. Take a screenshot of the release notes and post in the `#vsp-dst-engineers` channel on slack; ask for others to double-check that everything looks good and that there aren't any last-minute additions to the release that need to be included.
```
Comment (jamigibbs) [inline README.md:70]: The only other thing I think needs to be updated is that the screenshot of the release notes should be posted in #vsp-design-system-team (not just the engineers). The PO and designers will also want to be aware of releases and giving feedback if something doesn't look right.

Comment (Andrew565) [reply README.md:70]: Good call, thanks.

**Review by jamigibbs — APPROVED** [review]

**Review by harshil1793 — APPROVED** [review]

**Review by ataker — APPROVED** [review]

**Review by ataker — APPROVED** [review]

**Review by nickjg231 — APPROVED** [review]

**Review by harshil1793 — APPROVED** [review]

**Pull request merged**